mrmustard/physics/gaussian_integrals.py

```diff
@@ -258,12 +258,18 @@ def reorder_abc(Abc: tuple, order: Sequence[int]):
         The reordered ``(A,b,c)`` triple
     """
     A, b, c = Abc
+    dim_poly = len(c.shape) - 1
 
     if order:
+        if dim_poly > 0:
+            if type(order) == list:
+                order.extend(np.arange(len(order), len(order) + dim_poly).tolist())
+            elif type(order) == tuple:
+                order = order + tuple(np.arange(len(order), len(order) + dim_poly))
         A = math.gather(math.gather(A, order, axis=-1), order, axis=-2)
         b = math.gather(b, order, axis=-1)
-        if len(c.shape) == len(order):
-            c = math.transpose(c, order)
+        # if len(c.shape) == len(order):
+        #     c = math.transpose(c, order)
```

Comment:
Do these lines need to be uncommented or removed?

Reply:
I asked Filippo what the purpose of this is, and I don't think we came to a conclusion. Everything works is we remove them, and I cannot see why we would want to reorder the polynomials in the new ansatz. So they should probably be removed.